E2E tests failing as no Guacamole connection_uri output

```
>       endpoint = workspace_service["workspaceService"]["properties"]["connection_uri"]
E       KeyError: 'connection_uri'
```

Not sure why was't picked up as part of E2E tests - as I skipped them. :/

Need a way to check the Guacamole workspace service is up and running.

--- a/e2e_tests/test_workspace_services.py
+++ b/e2e_tests/test_workspace_services.py
@@ -59,7 +59,7 @@ async def test_create_guacamole_service_into_aad_workspace(setup_test_aad_worksp
 
 async def ping_guacamole_workspace_service(workspace_service_path, access_token, verify) -> None:
     workspace_service = await get_resource(f"/api{workspace_service_path}", access_token, verify)
-    endpoint = workspace_service["workspaceService"]["properties"]["connection_uri"]
+    endpoint = workspace_service["workspaceService"]["properties"]["admin_connection_uri"]
     await check_aad_auth_redirect(endpoint, verify)
 
 

--- a/templates/workspace_services/guacamole/porter.yaml
+++ b/templates/workspace_services/guacamole/porter.yaml
@@ -111,10 +111,17 @@ outputs:
     type: string
     applyTo:
       - install
+      - upgrade
   - name: web_apps_addresses
     type: string
     applyTo:
       - install
+      - upgrade
+  - name: admin_connection_uri
+    type: string
+    applyTo:
+      - install
+      - upgrade
 
 mixins:
   - terraform:
@@ -149,6 +156,7 @@ install:
       outputs:
         - name: authentication_callback_uri
         - name: web_apps_addresses
+        - name: admin_connection_uri
 
 upgrade:
   - terraform:
@@ -179,6 +187,7 @@ upgrade:
       outputs:
         - name: authentication_callback_uri
         - name: web_apps_addresses
+        - name: admin_connection_uri
 
 uninstall:
   - terraform:

--- a/templates/workspace_services/guacamole/terraform/outputs.tf
+++ b/templates/workspace_services/guacamole/terraform/outputs.tf
@@ -5,3 +5,7 @@ output "authentication_callback_uri" {
 output "web_apps_addresses" {
   value = jsonencode(data.azurerm_subnet.web_apps.address_prefixes)
 }
+
+output "admin_connection_uri" {
+  value = "https://${azurerm_linux_web_app.guacamole.default_hostname}/guacamole"
+}

